cms/articles/models.py

```diff
@@ -148,7 +148,7 @@ def release_with_versions(self, request: "HttpRequest", slug: str, version: int)
 class StatisticalArticlePage(BundledPageMixin, RoutablePageMixin, BasePage):  # type: ignore[django-manager-missing]
     """The statistical article page model.
 
-    Previously known as statistical bulletin, statistical analysis article, analysis page.
+    Previously known as statistical statistical_article, statistical analysis article, analysis page.
```

Comment:
```suggestion
    Previously known as statistical bulletin, statistical analysis article, analysis page.
```

at times, `s/bulletin/statistical_article/g` touches things we don't want it to :)

Reply:
`ctrl + d` gone wrong haha

Reply:
Reverted: [5d6c0f6](https://github.com/ONSdigital/dis-wagtail/pull/348/commits/5d6c0f6798b6ba1cf9f8c9dbaad144acf5c10563)